cve_bin_tool/cli.py

```diff
@@ -341,7 +341,10 @@ def main(argv=None):
             LOGGER.info(
                 f"There are {cve_scanner.products_with_cve} files with known CVEs detected"
             )
-        if cve_scanner.products_with_cve > 0:
+        if cve_scanner.products_with_cve > 0 or (
+            cve_scanner.products_with_cve == 0
```

Comment:
Can this ever be negative? If not why have this condition?
In case we allow all modes to create files with 0 CVEs this whole `if` condition can be dropped, will only need to put `LOGGER.info(f"Known CVEs in {affected_string}:")` under check below.

Reply:
No it won't be negative. Thanks for pointing it out.